### PR TITLE
Fix Safari styling

### LINKED ISSUE #68: Paper spinner not displaying properly in Safari see #56
### Description
Spinner just shows the initial small arc on Safari

### Expected outcome

Spinner arc should grow.

### Actual outcome

Spinner arc rotates but does not grow in length

### Live Demo
https://www.webcomponents.org/element/PolymerElements/paper-spinner

### Steps to reproduce

1. go to https://www.webcomponents.org/element/PolymerElements/paper-spinner in safari mac

### Browsers Affected

- [ ] Chrome
- [ ] Firefox
- [x] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

### Notes

This is the same issue as except it is showing up only in Safari #56

### PR DESCRIPTION
**Pull request by tpluscode**
Applying #82 by @aryashashank to the 3.0 codebase

Fixes #68

### CODE REVIEW

**Comment by tpluscode** [comment]
@aryashashank please confirm that you're fine with my reusing your commits in this PR (check the CLA bot comment above)

**Comment by aryashashank** [comment]
No problem at all.
I am happy to contribute.
:)

**Comment by pdesjardins90** [comment]
@keanulee could you merge this?

**Comment by keanulee** [comment]
I'm working on my [`safari`](https://github.com/PolymerElements/paper-spinner/tree/safari) branch because there's some changes I'd make to the PR. Also CLA is missing from this PR.

Have to run these changes through internal tests first, but I have higher priority things to do at the moment so it may be a while before I get to this.

**Comment by pdesjardins90** [comment]
Ok, but if I read correctly, the CLA is not checked because of @aryashashank's commits which he agreed to be used in this PR (see his), is there something he needs to do for the bot to change the check status?

**Comment by keanulee** [comment]
Oh yea, missed that bit about CLA. Thanks.

**Comment by pdesjardins90** [comment]
@keanulee thanks for the quick PR! keep up the good work

**Comment by tpluscode** [comment]
What was wrong with the original one?

File: paper-spinner-styles.js [hunk 1]
```diff
@@ -238,6 +238,19 @@ $_documentContainer.innerHTML = `<dom-module id="paper-spinner-styles">
         overflow: hidden;
       }
 
+      .circle-clipper .circle {
```
Comment (keanulee) [inline paper-spinner-styles.js]: In my branch, I combined most of these properties with `.spinner-layer::after` (like it was before).

I also had to add an empty `export {}` statement for some internal compatibility.

Comment (tpluscode) [reply paper-spinner-styles.js]: I see. Thank you for getting this to a finish line

**Pull request closed without merging**